pkg/vendir/directory/directory.go

```diff
@@ -89,6 +89,11 @@ func (d *Directory) Sync(syncOpts SyncOpts) (ctlconf.LockDirectory, error) {
 		if skipFetching {
```

Comment:
Hi @prembhaskal, I am open to renaming the flag, but it should not contain the word "cache", since we copy from the already fetched final content location and not a cache, i.e. a separate copy that is kept to increase performance. "Fetching" in this context means for me "to pull from the source location defined in the vendir.yml".

Reply:
hmm fine. I think for current review it is fine. But we can think of changing name as a refactoring PR.